frontend/src/libraries/explorer-wamp/contracts.ts

```diff
@@ -0,0 +1,58 @@
+import { ExplorerApi } from ".";
+
+export interface ContractInfo {
+  transactionHash: string;
+  timestamp: number | null;
+  accessKeys: Array<object>;
+  codeHash: string;
+}
+
+export default class ContractsApi extends ExplorerApi {
+  async getContractInfo(id: string) {
+    try {
+      const codeHash = await this.queryCodeHash(id);
+      if (codeHash !== "11111111111111111111111111111111") {
+        const [contractInfo, accessKeys] = await Promise.all([
+          this.call<any>("select", [
+            `SELECT transactions.block_timestamp, transactions.hash 
+            from transactions
```

Comment:
I noticed that you use `from` in the lowercase, let's use `FROM` to match the style of the rest of the SQL query in the future.